unlock-js/src/web3Service.js

```diff
@@ -411,13 +413,19 @@ export default class Web3Service extends UnlockService {
       blockTransaction.hash,
       contract,
       blockTransaction.data,
-      blockTransaction.to
+      blockTransaction.to,
+      'pending'
     )
   }
 
   /**
    * This refreshes a transaction by its hash.
    * It will only process the transaction if the filter function returns true
+   * There are at least 4 states for a transaction
```

Comment:
The changes below are no-op but an effort at clarifying what is going on. Not sure I succeeded!

Reply:
These descriptions do help, I was actually looking over this function earlier today in my investigation for #4470 and some of it was less than clear. Thanks for the 🚜 gardening!